main.py

```diff
@@ -0,0 +1,74 @@
+from datetime import date
+
+from fastapi import FastAPI, HTTPException
+import requests
+from pydantic import BaseModel
+
+import models
+from database import engine, SessionLocal
+
+app = FastAPI()
+models.Base.metadata.create_all(bind=engine)
+
+
+class PriceData(BaseModel):
+    symbol: str
+    open: float
+    high: float
+    low: float
+    price: float
+    volume: int
+    latest_trading_day: date
+    previous_close: float
+    change: float
+    change_percentage: float
+
+    class Config:
+        from_attributes = True
+
+
+db = SessionLocal()
+
+
+@app.get("/stock/quote")
+def get_stock_quote():
+    url = "https://alpha-vantage.p.rapidapi.com/query"
+
+    querystring = {"function": "GLOBAL_QUOTE", "symbol": "MSFT", "datatype": "json"}
+
+    headers = {
+        "X-RapidAPI-Key": "ADD-KEY",
+        "X-RapidAPI-Host": "alpha-vantage.p.rapidapi.com"
+    }
+
+    response = requests.get(url, headers=headers, params=querystring)
```

Comment:
what will happen if the request to alpha API times out or if there is any connection error, etc?
Let's handle the negative scenarios also i.e. exception handling for third-party API communication.